src/signals/incident/definitions/wizard-step-2-vulaan/overlast-op-het-water.ts

```diff
@@ -9,6 +9,13 @@ export const overlastOpHetWater = {
   locatie,
   dateTime: {
     meta: {
+      ifOneOf: {
+        subcategory: [
+          'olie-op-het-water',
```

Comment:
Can't find this category in the django admin.

Reply:
I think it's called 'Oppervlaktewaterverontreiniging' on prod, 'Olie op het water' on acc. I used the [categories as returned by the api](https://api.meldingen.amsterdam.nl/signals/v1/public/terms/categories)

Reply:
a I see, Django doens't search on slugs. Approved!